src/hgvs/variantmapper.py

```diff
@@ -483,8 +483,8 @@ def _replace_reference(self, var):
         else:
             pos = var.posedit.pos
 
-        seq_start = pos.start.base - 1
-        seq_end = pos.end.base
+        seq_start = pos.start.base + getattr(pos.start, 'offset', 0) - 1
```

Comment:
How about something like:

```
if var.type in "cnr":
    # assume pos have offset
else:
    # don't have offset

```

The subtext here is that I find getattr hard to read because it obfuscates the cases that we're trying to handle. I try mightily to avoid getattr with defaults.

Reply:
Same as above.